sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1313,131 +1313,178 @@ object functions {
   //////////////////////////////////////////////////////////////////////////////////////////////
 
   /**
-   * Computes the cosine inverse of the given value; the returned angle is in the range
-   * 0.0 through pi.
+   * @param e the value whose arc cosine is to be returned
+   * @return  cosine inverse of the given value in the range of 0.0 through pi,
```

Comment:
I'd use the same description here as above, for all of these functions. The one above looks better to me. Up to you whether you want to talk about the range of the return value or leave that to the Math.acos docs; it should just be consistent.

Also "cosine inverse" -> "inverse cosine", and make it clear as you do above that it's a synonym for arc cosine.

Reply:
Ok.

Then I will cut the part about the domains and ranges.

Reply:
I am not sure what you mean on _above_. Do you mean reverting this part of the change?

How about simply `@return the angle whose cosine is 'e'` and refer to java.lang.Math for further details?